### PR TITLE
Add goals for student - trivicks

### PR DESCRIPTION
**Pull request by trivicks**

### CODE REVIEW

**Comment by basavk** [comment]
Good code !!

**Comment by basavk** [comment]
comment 2

**Comment by basavk** [comment]
Hey @trivicks

**Comment by trivicks** [comment]
#1 closed

**Comment by trivicks** [comment]
Closes #1

**Comment by trivicks** [comment]
#2 closes.

File: README.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# Tutorial for Vikram
+
+* See videos
+* Try out commands
+* I love oatmeal
```
Comment (trivicks) [inline README.md]: really? @basavk  who loves oatmeal?

**Pull request merged**